Service action not working properly

When running a service action, the SDK is returning a non-related error:
```
vagrant@ansible ~/dev/katana  $ katana service action -c config.xml -p access_token="123" -n facebook
2017-05-05T07:24:05.407Z [INFO] Component initiated...
2017-05-05T07:24:07.527Z [ERROR] Middleware error running callback: facebook. Cannot get service schema. No mapping provided
2017-05-05T07:24:07.559Z [ERROR] Operation failed```
```

This is coming from very simple config and runner files:

config.xml
```xml
<?xml version="1.0" encoding="UTF-8"?>
<service xmlns="urn:katana:service" name="auth" version="1.0.0" http-base-path="/1.0/auth">
    </info>
    <engine runner="urn:katana:runner:node" path="./runner.js"/>
    <action name="facebook" http-method="post" http-input="form-data" http-path="/facebook" description="logs in via facebook">
        <param name="access_token" required="true" description="The access token of the user in fb"/>
        <schema type="entity" name="auth"/>
    </action>
    <schema-entity name="auth">
        <entity validate="true">
            <field name="token"/>
            <field name="refresh_token"/>
            <field name="expires" format="datetime"/>
        </entity>
    </schema-entity>
</service>
```

runner.js
```js
const Service = require('katana.sdk').Service;

const service = new Service();

service.action('facebook', (action) => {
    let graph = require('fbgraph');
    const token = action.getParam('access_token').getValue();
    graph.setAccessToken(token);
    graph.get('me?fields=id,email,location,picture', function (error, response) {
       action.log(response);
    });
    return action;
});

service.run();
```

--- a/sdk/action.js
+++ b/sdk/action.js
@@ -220,7 +220,15 @@ class Action extends Api {
    * @see https://github.com/kusanagi/katana-sdk-spec#actionset_returnmixed-value--action
    */
   setReturn(value) {
-    const serviceSchema = this.getServiceSchema(this.name, this.version);
+    let serviceSchema;
+
+    try {
+      serviceSchema = this.getServiceSchema(this.name, this.version);
+    } catch (e) {
+      this._return = value;
+
+      return this;
+    }
     const actionSchema = serviceSchema.getActionSchema(this._actionName);
 
     if (!actionSchema.hasReturn()) {
@@ -247,10 +255,19 @@ for action: "${this._actionName}"`
    * @return {boolean}
    */
   hasReturn() {
-    return this
-      .getServiceSchema(this._name, this._version)
-      .getActionSchema(this._actionName)
-      .hasReturn();
+
+    try {
+      return this
+        .getServiceSchema(this._name, this._version)
+        .getActionSchema(this._actionName)
+        .hasReturn();
+    } catch (e) {
+    }
+
+    // If we don't have a schema, return a sensible default
+    return false;
+  }
+
   /**
    *
    * @return {*}

